src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritCause/description.jelly

```diff
@@ -33,36 +34,25 @@
         </j:otherwise>
     </j:choose>
     <j:if test="${it.context.hasOthers()}">
-        <table cellpadding="2" cellspacing="0" border="0" style="margin-left: 15px;">
-            <tr>
-                <th colspan="2">
-                    ${%OtherTriggeredBuilds}
-                </th>
-            </tr>
+        <div class="jenkins-!-margin-bottom-1">
+            <div class="gt-other-builds__header jenkins-!-margin-bottom-1">
+                ${%OtherTriggeredBuilds}
+            </div>
             <j:forEach items="${it.context.getSortedOthers()}" var="other">
-                <tr>
+                <div class="gt-other-builds__line">
                     <j:choose>
-                        <j:when test="${other.hasBuild()}">
```

Comment:
Can you point out, where this change impact?

Reply:
when another build is referenced but that build has already been deleted the image was missing